packages/access-client/src/types.ts

```diff
@@ -64,6 +86,13 @@ export interface SpaceTableMetadata {
  * Access api service definition type
  */
 export interface Service {
+  access: {
+    authorize: ServiceMethod<
+      AccessAuthorize,
+      EncodedDelegation<[AccessSession]> | undefined,
+      Failure
+    >
```

Comment:
This doesn't return anything.

```suggestion
    authorize: ServiceMethod<AccessAuthorize, undefined, Failure>
```

Reply:
it does for tests